// Импортируем обработчики исключений (если они в другом файле)
import './exceptions.js'; // Убедитесь, что этот импорт правильный
// Функция, генерирующая ошибку
function throwError() {
    throw new Error("Это тестовая ошибка для uncaughtException!");
}
// Вызываем функцию для генерации ошибки
throwError();
// Здесь можно добавить остальной код приложения
